.github/workflows/auto-merge.yml

```diff
@@ -0,0 +1,115 @@
+name: Automerge PRs
+
+on:
+  schedule:
+    - cron: "*/10 * * * *" # Run every 10 minutes
+
+jobs:
+  merge:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Automerge PRs with "automerge" label created within the last month and are mergeable
+        uses: actions/github-script@v6
+        with:
+          github-token: ${{secrets.GITHUB_TOKEN}}
+          retries: 3
+          script: |
+            const { repo: { owner, repo } } = context;
```

Comment:
Can we create a javscript file and move the code there?

Reply:
Yes, we can https://github.com/actions/github-script#run-a-separate-file
Let me move this to the file.